templates/500.html

```diff
@@ -20,10 +20,21 @@
                 <div class="errorcontent">
                     <h1 class="lead">{{ _("Internal server error") }}</h1>
                     {% trans %}
-                    <p>This Zulip server is currently experiencing some technical difficulties. Sorry about that!</p>
-                    <p>The page will reload automatically soon after service is restored.</p>
-                    <p>If you'd like, you can <a href="mailto:{{ support_email }}">drop us a line</a> to let us know what happened.</p>
+                    <p>Your Zulip chat cannot be loaded because the server is experiencing technical difficulties.</p>
+                    <p> This page will reload automatically when service is restored.</p>
```

Comment:
The extra space before "This page" needs to be removed

Reply:
> The extra space before "This page" needs to be removed

Done